font_metro/init.lua

```diff
@@ -18,6 +18,7 @@ font_api.register_font(
 	  }
 );
```

Comment:
```suggestion
		widths = {
			[  0]=10, [ 32]=10, [ 33]=4,  [ 34]=7,  [ 35]=12, [ 36]=10, [ 37]=14, [ 38]=13, [ 39]=4,  [ 40]=6,
			[ 41]=6,  [ 42]=10, [ 43]=8,  [ 44]=4,  [ 45]=7,  [ 46]=4,  [ 47]=10, [ 48]=10, [ 49]=7,  [ 50]=10,
			[ 51]=10, [ 52]=10, [ 53]=10, [ 54]=10, [ 55]=10, [ 56]=10, [ 57]=10, [ 58]=4,  [ 59]=4,  [ 60]=10,
			[ 61]=9,  [ 62]=10, [ 63]=10, [ 64]=16, [ 65]=14, [ 66]=10, [ 67]=13, [ 68]=11, [ 69]=9,  [ 70]=9,
			[ 71]=13, [ 72]=11, [ 73]=4,  [ 74]=9,  [ 75]=12, [ 76]=9,  [ 77]=17, [ 78]=11, [ 79]=14, [ 80]=10,
			[ 81]=15, [ 82]=12, [ 83]=10, [ 84]=12, [ 85]=10, [ 86]=14, [ 87]=17, [ 88]=12, [ 89]=12, [ 90]=10,
			[ 91]=6,  [ 92]=10, [ 93]=6,  [ 94]=9,  [ 95]=9,  [ 96]=6,  [ 97]=9,  [ 98]=10, [ 99]=9,  [100]=10,
			[101]=10, [102]=7,  [103]=9,  [104]=10, [105]=4,  [106]=5,  [107]=10, [108]=5,  [109]=14, [110]=10,
			[111]=10, [112]=10, [113]=10, [114]=8,  [115]=8,  [116]=7,  [117]=10, [118]=9,  [119]=15, [120]=9,
			[121]=9,  [122]=8,  [123]=7,  [124]=4,  [125]=7,  [126]=12, [160]=6,  [161]=4,  [162]=10, [163]=10,
			[164]=14, [165]=12, [166]=4,  [167]=8,  [168]=8,  [169]=16, [170]=9,  [171]=11, [172]=10, [173]=6,
			[174]=16, [175]=8,  [176]=8,  [177]=8,  [178]=9,  [179]=9,  [180]=6,  [181]=10, [182]=9,  [183]=4,
			[184]=7,  [185]=7,  [186]=10, [187]=11, [188]=14, [189]=14, [190]=14, [191]=10, [192]=12, [193]=12,
			[194]=12, [195]=12, [196]=14, [197]=12, [198]=16, [199]=13, [200]=9,  [201]=9,  [202]=9,  [203]=9,
			[204]=5,  [205]=5,  [206]=6,  [207]=8,  [208]=12, [209]=11, [210]=12, [211]=12, [212]=12, [213]=12,
			[214]=14, [215]=10, [216]=14, [217]=10, [218]=10, [219]=10, [220]=10, [221]=12, [222]=9,  [223]=10,
			[224]=9,  [225]=9,  [226]=9,  [227]=9,  [228]=9,  [229]=9,  [230]=15, [231]=9,  [232]=10, [233]=10,
			[234]=10, [235]=10, [236]=4,  [237]=4,  [238]=4,  [239]=5,  [240]=10, [241]=10, [242]=10, [243]=10,
			[244]=10, [245]=10, [246]=10, [247]=10, [248]=10, [249]=10, [250]=10, [251]=10, [252]=10, [253]=9,
			[254]=10, [255]=9,  [256]=12, [257]=9,  [258]=12, [259]=9,  [260]=14, [261]=9,  [262]=11, [263]=9,
			[264]=11, [265]=9,  [266]=11, [267]=9,  [268]=11, [269]=9,  [270]=10, [271]=13, [272]=12, [273]=10,
			[274]=9,  [275]=10, [276]=9,  [277]=10, [278]=9,  [279]=10, [280]=9,  [281]=10, [282]=9,  [283]=10,
			[284]=11, [285]=9,  [286]=11, [287]=9,  [288]=11, [289]=9,  [290]=13, [291]=9,  [292]=11, [293]=10,
			[294]=13, [295]=11, [296]=8,  [297]=5,  [298]=6,  [299]=4,  [300]=8,  [301]=5,  [302]=4,  [303]=4,
			[304]=4,  [305]=4,  [306]=12, [307]=8,  [308]=10, [309]=5,  [310]=12, [311]=10, [312]=10, [313]=9,
			[314]=6,  [315]=9,  [316]=5,  [317]=9,  [318]=7,  [319]=9,  [320]=7,  [321]=10, [322]=7,  [323]=11,
			[324]=10, [325]=11, [326]=10, [327]=11, [328]=10, [329]=12, [330]=11, [331]=10, [332]=12, [333]=10,
			[334]=12, [335]=10, [336]=12, [337]=10, [338]=18, [339]=16, [340]=11, [341]=8,  [342]=12, [343]=8,
			[344]=11, [345]=8,  [346]=10, [347]=8,  [348]=10, [349]=8,  [350]=10, [351]=8,  [352]=10, [353]=8,
			[354]=12, [355]=7,  [356]=12, [357]=9,  [358]=12, [359]=7,  [360]=10, [361]=10, [362]=10, [363]=10,
			[364]=10, [365]=10, [366]=10, [367]=10, [368]=10, [369]=10, [370]=10, [371]=10, [372]=15, [373]=15,
			[374]=12, [375]=9,  [376]=12, [377]=10, [378]=8,  [379]=10, [380]=8,  [381]=10, [382]=8,  [383]=5,
			[894]=4,  [913]=14, [914]=10, [915]=9,  [916]=14, [917]=9,  [918]=10, [919]=11, [920]=14, [921]=4,
			[922]=12, [923]=14, [924]=17, [925]=11, [926]=10, [927]=14, [928]=11, [929]=10, [931]=10, [932]=12,
			[933]=12, [934]=12, [935]=12, [936]=12, [937]=14, [945]=10, [946]=10, [947]=9,  [948]=10, [949]=9, 
			[950]=9,  [951]=10, [952]=10, [953]=5,  [954]=10, [955]=9,  [956]=10, [957]=9,  [958]=9,  [959]=10,
			[960]=11, [961]=10, [962]=9,  [963]=10, [964]=10, [965]=10, [966]=12, [967]=9,  [968]=12, [969]=12,
			[1026]=14,[1028]=13,[1033]=17,[1034]=17,[1035]=14,[1037]=11,[1039]=11,[1040]=14,[1041]=10,[1042]=10,
			[1043]=10,[1044]=12,[1045]=9, [1046]=18,[1047]=10,[1048]=11,[1049]=11,[1050]=12,[1051]=11,[1052]=17,
			[1053]=11,[1054]=14,[1055]=11,[1056]=10,[1057]=13,[1058]=12,[1059]=12,[1060]=12,[1061]=12,[1062]=12,
			[1063]=10,[1064]=16,[1065]=17,[1066]=13,[1067]=13,[1068]=10,[1069]=13,[1070]=17,[1071]=12,[1072]=9, 
			[1073]=10,[1074]=9, [1075]=8, [1076]=11,[1077]=10,[1078]=16,[1079]=9, [1080]=11,[1081]=11,[1082]=10,
			[1083]=10,[1084]=13,[1085]=10,[1086]=10,[1087]=10,[1088]=10,[1089]=9, [1090]=10,[1091]=9, [1092]=16,
			[1093]=9, [1094]=10,[1095]=9, [1096]=14,[1097]=15,[1098]=11,[1099]=12,[1100]=9, [1101]=10,[1102]=13,
			[1103]=11,[8216]=4, [8217]=4, [8220]=7, [8221]=7, [8364]=15
		},
	}
);
```

Reply:
fonts are created w/ a script to read the diff easier i think this is a bad idea

Reply:
Oh, that's true... Then maybe add a comment about that. (and in the other file)